### ISSUE TITLE
Open Source community page

### ISSUE THREAD

**Issue by ritikrajdev** [MicrosoftStudentChapter/MSC-Website#36]
A page which describes our open source community should be made. In this page mlsc's open source community should be described and events held over there should be briefed.

**Comment by ShreyaChopra13** [comment]
I would like to take this up

**Comment by aarushi353** [comment]
I will take up this issue

**Comment by Anureet123** [comment]
I would like to take up this issue

**Comment by harikanani** [comment]
Any updates on this issue?
What's your progress guys! @aarushi353 @ShreyaChopra13

**Comment by aarushi353** [comment]
Yes @harikanani ,We are currently working on the issue.The design is completed and we are working over the front end part.
